python/location/lbs/activity/audioset/dataprocessing/audio_processing.py

```diff
@@ -0,0 +1,262 @@
+# This Python file is an extended/modified version of Jerry Wu's audio_prep.py
```

Comment:
I actually don't think we need to reference Jerry's code here. I had originally thought Jerry had already committed his code, but if he hasn't, I think it's okay for you to just push it in as is and mention it in your pull request that you had adapted code from Jerry.

Be sure to include a module docstring, see example below:

"""A one line summary of the module or program, terminated by a period.

Leave one blank line. The rest of this docstring should contain an
overall description of the module or program. Optionally, it may also
contain a brief description of exported classes and functions and/or usage
examples.

Typical usage example:

foo = ClassFoo()
bar = foo.FunctionBar()
"""